themes/janeswalk/js/components/pages/WalkPage/WalkMap.jsx

```diff
@@ -25,10 +24,12 @@ export default class WalkMap extends React.Component {
     const mapOptions = {
       center: locationLatLng,
       zoom: 13,
+      scrollwheel: false,
       backgroundColor: '#d7f0fa',
+
     };
 
-    const googleMap = new google.maps.Map(ReactDOM.findDOMNode(this),mapOptions);
+    const googleMap = new google.maps.Map(React.findDOMNode(this),mapOptions);
```

Comment:
I usually write this the way you have, but React 0.14 updated the way refs behave, so they always return native DOM nodes. This can be handy for quickly grabbing a DOM node without the `React.findDOMNode` call, but the main reason I prefer it is it lets you expand the component later if you want to move the map somewhere inside.. e.g.

```
<section>
  <h1>Welcome to my sweet map!</h1>
  <div className="walkMap" ref="maptarget" />
</section>
```
Then when you render to it, just say `new google.maps.Map(this.refs.maptarget, mapOptions)`